CHANGELOG.md

```diff
@@ -502,6 +605,16 @@ Additions to existing modules
   HomoProduct  n A = HomoProduct′ n (const A)
   ```
 
+* In `Data.Sum.Relation.Binary.LeftOrder` :
+  ```agda
+  ⊎-<-wellFounded : WellFounded ∼₁ → WellFounded ∼₂ → WellFounded (∼₁ ⊎-< ∼₂)
+  ```
+
+* in `Data.Sum.Relation.Binary.Pointwise` :
+  ```agda
+  ⊎-wellFounded : WellFounded ∼₁ → WellFounded ∼₂ → WellFounded (Pointwise ∼₁ ∼₂)
+  ```
+
```

Comment:
Approve

Reply:
Weird! CHANGELOG merge hell strikes again. Will review.

Reply:
to my eyes, the names are the same?